tables/translated/kRO_english/attendance_rewards.txt

```diff
@@ -0,0 +1,22 @@
+start 20210504
```

Comment:
This file must be deleted. This is an extra file, it is individual for each server.